manifests/agent.pp

```diff
@@ -395,7 +397,7 @@
     ensure  => present,
     owner   => $agent_config_owner,
     group   => $agent_config_group,
-    mode    => '0644',
+    mode    => '0664',
```

Comment:
can you explain why the change is needed?

Reply:
This is needed because Puppet and Windows file permissions don't play nice. If I leave it at 644 Puppet will change it every run (but nothing will actually happen)